specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/Ocr.json

```diff
@@ -118,9 +118,6 @@
         "parameters": [
           {
             "$ref": "#/parameters/ImageUrl"
-          },
```

Comment:
Was an SDK released for this already? These are breaking changes on a stable version of the API

Reply:
Yes. There's was an SDK released already. But since there's a recent update from backend service, we want to remove the parameter.

Do you know which process we should follow for this kind of breaking change?

Reply:
@toothache you will need to release new major versions for all the released SDKs so as to not break existing customers.
This will need an API review too

Reply:
@dsgouda we have reviewed the GA version API, in which mode parameter is removed, with API review board before. But we're also sending mail to confirm it.